README.rst

```diff
@@ -132,15 +132,15 @@ project that you might want to use:
 - `ptpython and ptipython <https://pypi.org/project/ptpython/>`
 - `xonsh <https://xon.sh/>`
 
-Ignoring commits with git blame.ignoreRevsFile
+Ignoring commits with the git blame.ignoreRevsFile
 ==============================================
```

Comment:
That is technically incorrect, `blame.ignoreRevsFile` is an option, thus it should be treated like a proper name and not a common name. Plus the RST syntax is now invalid with the underline.